mind_matter_api/api/surveys.py

```
@@ -26,124 +28,198 @@ def get_surveys():
             data = SurveyResponseSchema().load(request.get_json())
         except ValidationError as err:
             return jsonify(err.messages), 400
+
         data['survey_id'] = survey_id
         r = svc.create_response(data)
         return jsonify(SurveyResponseSchema().dump(r)), 201
 
     @app.route('/responses/<int:response_id>', methods=['PUT'])
     def update_response(response_id):
+        user_id = get_authenticated_user_id_or_abort()
+        if isinstance(user_id, tuple): return user_id
```

Ruff (0.8.2)

Multiple statements on one line (colon)

(E701)